### PR TITLE
updated invoice details

### PR DESCRIPTION
**Pull request by MounikaSharan**

### CODE REVIEW

File: src/app/features/invoice_details/dialogs/add/add-invoice-details.dialog.component.ts [hunk 1]
```diff
@@ -11,10 +11,14 @@ import {InvoiceDetailsDataService } from '../../services/invoice-details-data.se
...
 
 export class AddInvoiceDetailsDialogComponent {
+  clients:any
+  statuses:any
  
   constructor(public dialogRef: MatDialogRef<AddInvoiceDetailsDialogComponent>,
               @Inject(MAT_DIALOG_DATA) public data: InvoiceDetails,
               public dataService: InvoiceDetailsDataService) {
+                this.statuses = this.dataService.getStatuses()
+                this.clients = this.dataService.getClients()
```
Comment (yogeshrajput335) [inline src/app/features/invoice_details/dialogs/add/add-invoice-details.dialog.component.ts:22]: remove extra line

File: src/app/features/invoice_details/invoice-details-datasource.ts [hunk 2]
```diff
@@ -75,12 +75,16 @@ export class InvoiceDetailsDataSource extends DataSource<InvoiceDetails> {
...
       switch (this._sort.active) {
         case 'id': [propertyA, propertyB] = [a.id, b.id]; break;
-        case 'invoiceId': [propertyA, propertyB] = [a.invoiceId, b.invoiceId]; break;
-        case 'employeeId': [propertyA, propertyB] = [a.employeeId, b.employeeId]; break;
-        case 'projectId': [propertyA, propertyB] = [a.projectId, b.projectId]; break;
-        case 'perHourCost': [propertyA, propertyB] = [a.perHourCost, b.perHourCost]; break;
-        case 'totalHours': [propertyA, propertyB] = [a.totalHours, b.totalHours]; break;
-        case 'totalCost': [propertyA, propertyB] = [a.totalCost, b.totalCost]; break;
+        case 'invoiceNo': [propertyA, propertyB] = [a.invoiceNo, b.invoiceNo]; break;
+        // case 'createdDate': [propertyA, propertyB] = [a.createdDate, b.createdDate]; break;
```
Comment (yogeshrajput335) [inline src/app/features/invoice_details/invoice-details-datasource.ts]: remove commented code

File: src/app/features/invoice_details/dialogs/add/add-invoice-details.dialog.component.ts (hunk 1 above)
Comment (MounikaSharan) [reply src/app/features/invoice_details/dialogs/add/add-invoice-details.dialog.component.ts]: removed extra

**Review by yogeshrajput335 — APPROVED** [review]
Approved

File: src/app/features/invoice_details/invoice-details-datasource.ts (hunk 2 above)
Comment (MounikaSharan) [reply src/app/features/invoice_details/invoice-details-datasource.ts]: done

**Pull request merged**